src/App/Service/Github.php

```diff
@@ -252,6 +261,45 @@ public function getReviews(string $org, string $repository = ''): array
         return $result;
     }
 
+    public function getReviewsContributions(string $organizationId, ReviewFilter $reviewFilter, string $repository = ''): ReviewContributionsByOrganization
+    {
+        $graphQLQuery = '{
+            user(login: "' . $reviewFilter->getReviewer() . '") {
+                contributionsCollection(organizationID:"' . $organizationId . '" from: "' . $reviewFilter->getStartDate()->format('Y-m-d') . 'T00:00:00" to: "' . $reviewFilter->getEndDate()->format('Y-m-d') . 'T23:59:59") {
+                    pullRequestReviewContributionsByRepository {
+                        contributions(last: 100) {
+                            totalCount
+                            nodes {
+                                occurredAt
+                                pullRequestReview {
+                                    state
+                                    pullRequest {
+                                        author {
+                                            login
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                        repository {
+                            name
+                        }
+                    }
+                }
+            }
+          }';
```

Comment:
It's just a suggestion but usually in the JS world, we're splitting these queries into different files in order to avoid having big queries inside the logic of the code, wdyt about this?